### ISSUE TITLE
Explore integrating Experiment Overview and Results views

### ISSUE THREAD

**Issue by razvanpapadopol** [Automattic/abacus#770]
In fact, I was thinking we could even integrate the two, maybe with an summary/detailed switchable overview.

_Originally posted by @jessie-ross in https://github.com/Automattic/abacus/issues/742#issuecomment-1170556395_

**Comment by razvanpapadopol** [comment]
We've got the first report of an UX potential issue after #766 and it's about adding metrics on the landing page of a non-staging experiment: p1662048465028359-slack-C7HH3V5AS

As a first step towards integration, can we consider adding the "Assign Metric" button above or below the Metric Assignments Results table? Making the [Dialog](https://github.com/Automattic/abacus/blob/master/src/components/experiments/single-view/overview/MetricAssignmentsPanel.tsx#L242-L370) reusable should be straight-forward.

**Comment by jessie-ross** [comment]
> As a first step towards integration, can we consider adding the "Assign Metric" button above or below the Metric Assignments Results table?

Yeah I was thinking about that, but I think Experimenters will learn soon enough to not make it worthwhile.

**Comment by aaronyan** [comment]
Adding another 'Assign Metric' button is pretty low cost imo, so I'd be open to it!

**Comment by aaronyan** [comment]
What would be the best way to design an integrated Overview + Results view? Should one person mock up a design, and then we iterate from there?

**Comment by jessie-ross** [comment]
> Adding another 'Assign Metric' button is pretty low cost imo

Well, the modal is embedded in the overview tab so it would need to be extracted, and if we have an extracted modal we would need to create an abstraction around them so things don't get out of hand.

**Comment by jessie-ross** [comment]
The alternative here is the possibility of the fully integrated page, which I think will bring benefits.

> Well, the modal is embedded in the overview tab so it would need to be extracted, and if we have an extracted modal we would need to create an abstraction around them so things don't get out of hand.

I guess this stuff should happen anyway.

**Comment by razvanpapadopol** [comment]
> Experimenters will learn soon enough to not make it worthwhile.

This is true, but adding shortcuts can help with that process as well. Another dirt cheap alternative would be to add a link button that redirects to the correct section of adding/viewing metrics (which happens to be on the Overview tab right now).

> What would be the best way to design an integrated Overview + Results view? Should one person mock up a design, and then we iterate from there?

We can probably start by just joining the two views into one, with the Overview panels collapsed when the experiment is not in staging and with the Results hidden when it is. As I've [mentioned](https://github.com/Automattic/abacus/issues/742#issuecomment-1171457420) before, I really like the IA and simplicity of the current UI so it would be hard to come up with radical changes.
